src/article-crawler.js

```diff
@@ -18,43 +19,56 @@ module.exports = async function (sourceConfigs) {
 
 			const linkSelector = page['link-selector']
 			const articleUrls = await browserPage.$$eval(linkSelector, (elements) => elements.map((element) => element.href))
+			for (const articleUrl of articleUrls.slice(0,3)) {
```

Comment:
I think length 3 should be passed as parameter. I am thinking if this library can be made generic for public to use.